test/test-migrate-fixup.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+. "test/test-migrate-fixtures.sh"
+. "test/testlib.sh"
+
+begin_test "migrate import (--fixup)"
```

Comment:
For greater test coverage, it might be good to have a test involving a nested file and a nested .gitattributes file. That's already being tested in unit tests for each individual part utilized in the fixup, but I think it would be worthwhile to have a test here involving all those parts working together

Reply:
Good call, I caught this in https://github.com/git-lfs/git-lfs/pull/3114/commits/853e7fce3b2b7209914466967c95fc6440ef1c2c.